spa/src/views/claims/claimsView.tsx

```diff
@@ -40,11 +41,8 @@ export function ClaimsView(props: ClaimsProps) {
             const remoteError = e as RemoteError;
             if (remoteError) {
 
-                if (remoteError.getStatus() === 401) {
+                if (ErrorHandler.isSessionExpiredError(remoteError)) {
```

Comment:
Maybe `isSesssionExpired` and `isAccessTokenExpired` could be methods on the `RemoteError` class? The code would be cleaner then.